### DISCUSSION TITLE
Option to adjust osu!mania button width

### DISCUSSION THREAD

**Discussion by Coldtrigon66** [ppy/osu#20795] [Ideas] [answered]
So osu!mania on my phone looks like this ![Screenshot_20221017-103516.png](https://user-images.githubusercontent.com/66516725/196131512-3d11f628-dcc7-4ff2-9087-07b30efbc27f.png); it's straight up unplayable because of how small the buttons are. It would help if I could adjust how wide the overlay is

**Comment by integerrr** [comment, marked as answer]
#20624
